core/lib/resources/presence/index.js

```diff
@@ -33,7 +33,7 @@ Presence.prototype.setup = function() {
   var self = this;
 
   this.manager = new PresenceManager(this.name, this.options.policy, Presence.sentry);
-  this.manager.on('user_online', function(userId, userType, userData) {
+  this.manager.on('user_online', function(userId, userType, userData, data) {
```

Comment:
All of them? Like on the whole app? I can do it but I want to understand
the scope.

On Wednesday, May 27, 2015, Vanchi Koduvayur <notifications@github.com>
wrote:

> In core/lib/resources/presence/index.js
> <https://github.com/zendesk/radar/pull/136#discussion_r31156955>:
>
> > @@ -33,7 +33,7 @@ Presence.prototype.setup = function() {
> >    var self = this;
> >
> >    this.manager = new PresenceManager(this.name, this.options.policy, Presence.sentry);
> > -  this.manager.on('user_online', function(userId, userType, userData) {
> > +  this.manager.on('user_online', function(userId, userType, userData, data) {
>
> suggestion: Branch off lflorio/presence_states. Fix all this long
> parameter problems. Create a new PR into lflorio/presence_states. Get a +1
> on this PR and that PR. Then merge correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/radar/pull/136/files#r31156955>.
>


-- 
Lucas Florio
Software Engineer
Zendesk <http://www.zendesk.com>